static/app/views/settings/account/notifications/fields2.tsx

```diff
@@ -18,6 +19,7 @@ export const NOTIFICATION_SETTING_FIELDS: Record<string, NotificationSettingFiel
       ['always', t('On')],
       ['never', t('Off')],
     ],
+    help: 'Enable this to receive alert notifications sent from team alerts.',
```

Comment:
These should be translated

Reply:
I also think this text is a little strange, I can see it's straight from the design but we don't have an outward facing concept of team alerts (aside from what we're building which will live on a different page). Maybe something like "Enable this to receive notifications sent from project alerts"?